lib/cpp/src/thrift/TApplicationException.h

```diff
@@ -21,6 +21,7 @@
 #define _THRIFT_TAPPLICATIONEXCEPTION_H_ 1
 
 #include <thrift/Thrift.h>
+#include <stdint.h>
```

Comment:
Figured it out; wasn't able to build with VS2010 due to changes made in THRIFT-2835.  I made the necessary changes and ensured I could build on windows and on linux (to a point... I am still running into THRIFT-3937 on my linux system).